src/OpenLoco/Map/TileManager.cpp

```diff
@@ -680,6 +682,41 @@ namespace OpenLoco::Map::TileManager
         return surroundingWaterTiles;
     }
 
+    // 0x00469B1D
+    uint16_t countSurroundingDesertTiles(const Pos2& pos)
+    {
+        // Search a 10x10 area centred at pos.
+        // Initial tile position is the top left of the area.
+        auto initialTilePos = Map::TilePos2(pos) - Map::TilePos2(5, 5);
+
+        uint16_t surroundingDesertTiles = 0;
+
+        for (const auto& tilePos : TilePosRangeView(initialTilePos, initialTilePos + Map::TilePos2{ 10, 10 }))
+        {
+            if (!Map::validCoords(tilePos))
+                continue;
+
+            auto tile = get(tilePos);
+            auto* surface = tile.surface();
+            // Deserts can't have water!
```

Comment:
Is this true? They generally have oases (e.g. in Oil Oasis scenario).

Reply:
Hmm should i rename to built in dry areas?

Reply:
I think i'll keep it as is.